### ISSUE TITLE
Accessing In-App Frontlight Dialog (not Android native brightness popup)

### ISSUE THREAD

**Issue by dcruzbosco** [koreader/koreader#6815]
* KOReader version: 2020.10.1
* Device: MiPad 1

#### Issue
Since version 2020.09, accessing the Frontlight dialog opens native Android brightness popup.
I am interested in knowing how I can go back to the old in-app Frontlight dialog (widget) that allows step-by-step brightness adjustment. 
Is there a setting or property to be added in settings.lua for example, that would allow me to switch back to the old brightness adjustment popup?

**Comment by pazos** [comment]
Nope, you can't. The good old frontlight widget is not available for a very good reason: it was a PITA to work with and didn't support per device overrides.

>  step-by-step brightness adjustment.

This is doable with the android dialog too. But needs to be done :)

**Comment by dcruzbosco** [comment]
Thanks! This helps a lot!

**Comment by pazos** [comment]
@dcruzbosco: I'm going to update the ticket. Patches welcome!
